reserved-account-keys/Cargo.toml

```diff
@@ -31,6 +35,7 @@ solana-sdk-ids = { workspace = true }
 
 [dev-dependencies]
 solana-message = { workspace = true }
+solana-pubkey = { workspace = true, default-features = false, features = ["rand"] }
```

Comment:
This is only using `new_unique()` and not `rand`, so this feature can get changed to `std` instead